package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-pay-report-generator",
-  "version": "1.1.5",
-  "description": "Generate MI and suppressed payments reports",
+  "version": "1.2.0",
+  "description": "Generate payments reports for pay-web",
```

Comment:
Bit nitpicky, but I'd remove reference to which service is the current consumer.  This service doesn't care who picks up it's reports, if anyone.

What about `Generate payment reports`? or `Generator reports from Payment Hub processing`?  Or something better

Reply:
Generalised description